hibike/virtual_device.py

```diff
@@ -26,11 +26,22 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-d', '--device', required=True, help='device type')
     parser.add_argument('-p', '--port', required=True, help='serial port')
+    parser.add_argument('-v', '--verbose',
+                        help='print messages when sending and receiving packets',
+                        action="store_true")
     args = parser.parse_args()
 
+    def verbose_log(fmt_string, *fmt_args):
```

Comment:
I like this.
we should think about pulling this out into some general utilities shared between hibike + runtime. although that may be too much effort.